src/user.js

```diff
@@ -10,6 +10,16 @@ mongoose.connect('mongodb://localhost/users', { useMongoClient: true });
 
 const UserSchema = new mongoose.Schema({
   // TODO: fill in this schema
+  username: {
+    type: String,
+    required: true,
+    unique: true,
+  },
+  passwordHash: {
+    type: String,
+    required: true,
+    unique: true,
```

Comment:
I wouldn't actually enforce passwords to be unique. It's very possible for two people to pick the same password, especially since most people pick from a pool of very common passwords.